src/com/hotmail/kalebmarc/textfighter/main/Enemy.java

```diff
@@ -109,6 +110,11 @@ void dealDamage() {
         Health.takeDamage(damage);
     }
 
+    void returnDamage(int damage) {

```

Comment:
this naming is a bit confusing. "return" makes me think it's going to return a value; I would recommend changing the name, or adding a comment to explain it a little better. In this specific situation, I think it would make sense to name it `setDamageDealt`

Reply:
Yes, I am already aware of this issue at this moment. Still, thanks for pointing it out!

Reply:
just wanted to note that this would be considered a `setter`! Just like `getter`, but.. well, you know